src/command_modules/azure-cli-interactive/azclishell/az_completer.py

```diff
@@ -88,11 +86,7 @@ def _get_weight(val):
             priority = ' '  # a space has the lowest ordinance
         return priority + val.text
 
-    completions = []
-    for comp in gen:
-        if comp.text not in BLACKLISTED_COMPLETIONS:
-            completions.append(comp)
-    return sorted(completions, key=_get_weight)
+    return sorted(list(completions_gen), key=_get_weight)
```

Comment:
You don't have to create a list from `completions_gen`. The `sorted` function will create a new list from the iterable.
Reference: https://docs.python.org/3/library/functions.html#sorted